velox/docs/configs.rst

```diff
@@ -111,6 +111,10 @@ Generic Configuration
      - true
      - Whether to enable caches in expression evaluation. If set to true, optimizations including vector pools and
        evalWithMemo are enabled.
+   * - max_split_preload_per_driver
```

Comment:
Perhaps, start a new section Table Scan and move it there. I wonder if there are other TableScan-specific configs that are missing from the docs.

Reply:
Sure, there should be only two configs for scan, `table_scan_getoutput_time_limit_ms` and this one.

Reply:
Got it. It would be great to document table_scan_getoutput_time_limit_ms in a follow-up PR.

Reply:
https://github.com/facebookincubator/velox/pull/8100, thanks.